.travis.yml

```diff
@@ -1,17 +1,7 @@
 language: python
-
 python:
-  - "3.6"
-  - "3.7"
   - "3.8"
-  - "nightly"
-
-matrix:
-  allow_failures:
-    - python: "nightly"
-
```

Comment:
What about leaving the nightly test? This just tests against future releases. If it fails that is allowed, but if it is passing then we know there is no concern about updating to a major release

Reply:
I don't think anyone is going to be watching if the nightly test passes or fails. And python in general is pretty conservative with new updates. I believe they like to stay backwards compatible as much as possible. Leaving this in increases the amount of time one has to wait for the CI to run on their PR. And if the nightly test does start failing it could be out of our control to fix. For example: The failure could be caused due to a library we are using. I would suggest leaving it out.